PublicDateAtUSN: 2023-07-06
Candidate: CVE-2023-29824
PublicDate: 2023-07-06 21:15:00 UTC
References:
 http://www.square16.org/achievement/cve-2023-29824/
 https://github.com/scipy/scipy/pull/15013
 https://ubuntu.com/security/notices/USN-6226-1
 https://www.cve.org/CVERecord?id=CVE-2023-29824
Description:
 ** DISPUTED ** A use-after-free issue was discovered in Py_FindObjects()
 function in SciPy versions prior to 1.8.0. NOTE: the vendor and discoverer
 indicate that this is not a security issue.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
 https://github.com/scipy/scipy/issues/14713
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H [9.8 CRITICAL]
 cisa-adp: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H [9.8 CRITICAL]

Patches_scipy:
upstream_scipy: needs-triage
trusty_scipy: ignored (end of standard support)
xenial_scipy: ignored (end of standard support)
bionic_scipy: ignored (end of standard support)
focal_scipy: not-affected (disputed)
esm-infra/focal_scipy: DNE (focal was not-affected [disputed])
esm-apps/focal_scipy: released (1.3.3-3ubuntu0.1~esm1)
jammy_scipy: not-affected (1.8.0-1exp2ubuntu1)
esm-apps/jammy_scipy: not-affected (1.8.0-1exp2ubuntu1)
kinetic_scipy: not-affected (1.8.1-10)
lunar_scipy: not-affected (1.10.1-1)
mantic_scipy: not-affected (1.10.1-2)
noble_scipy: not-affected (1.10.1-2)
esm-apps/noble_scipy: not-affected (1.10.1-2)
devel_scipy: not-affected (1.10.1-2)
